src/main/java/uk/gov/hmcts/reform/sscs/model/partiesnotified/PartiesNotified.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.sscs.model.partiesnotified;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+public class PartiesNotified {
```

Comment:
We already have a PartiesNotified class why was this added and the other not removed or altered?

Reply:
Where?